medusa/post_processor.py

```diff
@@ -362,17 +364,14 @@ def _combined_file_operation(self, file_path, new_path, new_base_name, associate
             return
 
         # base name with file path (without extension and ending dot)
-        old_base_name = file_path.rpartition('.')[0]
+        old_base_name = os.path.splitext(file_path)[0]
         old_base_name_length = len(old_base_name)
 
         for cur_file_path in file_list:
             # remember if the extension changed
             changed_extension = None
             # file extension without leading dot (for example: de.srt)
             extension = cur_file_path[old_base_name_length + 1:]
```

Comment:
extension here for the nfo = u''

so when hardlinking the file, its moving without extension:

Quantico\Season 02\Quantico.S02.E11.ZRTORCH.720p.HDTV.x264. 

```
2017-02-22 09:44:42 DEBUG    Thread-27 :: [51ff22a] Deleting file C:\Users\Fernando\Documents\PP\Quantico.S02E11.720p.HDTV.X264-DIMENSION\quantico.211.720p-dimension.sfv
2017-02-22 09:44:42 DEBUG    Thread-27 :: [51ff22a] Moving file from C:\Users\Fernando\Documents\PP\Quantico.S02E11.720p.HDTV.X264-DIMENSION\Quantico.S02E11.720p.HDTV.X264-DIMENSION.mkv to C:\Users\Fernando\Documents\Séries\Quantico\Season 02\Quantico.S02.E11.ZRTORCH.720p.HDTV.x264.mkv 
2017-02-22 09:44:42 DEBUG    Thread-27 :: [51ff22a] Moving file from C:\Users\Fernando\Documents\PP\Quantico.S02E11.720p.HDTV.X264-DIMENSION\quantico.211.720p-dimension.nfo to C:\Users\Fernando\Documents\Séries\Quantico\Season 02\Quantico.S02.E11.ZRTORCH.720p.HDTV.x264. 
2017-02-22 09:44:42 DEBUG    Thread-27 :: [51ff22a] Moving file from C:\Users\Fernando\Documents\PP\Quantico.S02E11.720p.HDTV.X264-DIMENSION\Quantico.S02E11.720p.HDTV.X264-DIMENSION.pt-BR.srt to C:\Users\Fernando\Documents\Séries\Quantico\Season 02\Quantico.S02.E11.ZRTORCH.720p.HDTV.x264.pt-BR.srt 
```

![image](https://cloud.githubusercontent.com/assets/2620870/23215879/b081da1a-f8f2-11e6-826d-152b402e640c.png)

![image](https://cloud.githubusercontent.com/assets/2620870/23215898/c386c77e-f8f2-11e6-9900-ea496a5de275.png)